doc/mgr/restful.rst

```diff
@@ -77,6 +77,103 @@ If the port is not configured, *restful* will bind to port ``8003``.
 If the address it not configured, the *restful* will bind to ``::``,
 which corresponds to all available IPv4 and IPv6 addresses.
 
+.. _creating-an-api-user:
+
+Creating an API User
+-----------------------
+
+To create an API user, please run the following command::
+
+  ceph restful create-key <username>
+
+Replace ``<username>`` with the desired name of the user. For example, to create a user named
+``api``::
+
+  root@system1:/home# ceph restful create-key api
```

Comment:
could you remove the prompt so the command line example look like
```
$  ceph restful create-key api
52dffd92-a103-4a10-bfce-5b60f48f764e
```

Reply:
@tchaikov Sure :)